androidApp/build.gradle.kts

```diff
@@ -82,6 +82,7 @@ dependencies {
     implementation(libs.mapbox.turf)
     implementation(libs.okhttp)
     implementation(libs.playServices.location)
+    implementation(libs.androidx.lifecycle.runtime.testing)
```

Comment:
Should this be an `androidTestImplementation` dependency so it's not available in the `main` source root (and so it's not included in the dependency list)?